sdk/eventgrid/azure-messaging-eventgrid/swagger/README.md

```diff
@@ -81,33 +81,33 @@ custom-types: CloudEvent,EventGridEvent,AcsRouterCommunicationError,AcsMessageCh
-- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/1b0e82df2f7f953796af2aee0372e34e00baa2d2/specification/eventgrid/data-plane/Microsoft.MachineLearningServices/stable/2018-01-01/MachineLearningServices.json
-- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/1b0e82df2f7f953796af2aee0372e34e00baa2d2/specification/eventgrid/data-plane/Microsoft.Cache/stable/2018-01-01/RedisCache.json
-- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/1b0e82df2f7f953796af2aee0372e34e00baa2d2/specification/eventgrid/data-plane/Microsoft.Web/stable/2018-01-01/Web.json
-- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/1b0e82df2f7f953796af2aee0372e34e00baa2d2/specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json
-- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/1b0e82df2f7f953796af2aee0372e34e00baa2d2/specification/eventgrid/data-plane/Microsoft.PolicyInsights/stable/2018-01-01/PolicyInsights.json
-- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/1b0e82df2f7f953796af2aee0372e34e00baa2d2/specification/eventgrid/data-plane/Microsoft.ContainerService/stable/2018-01-01/ContainerService.json
-- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/1b0e82df2f7f953796af2aee0372e34e00baa2d2/specification/eventgrid/data-plane/Microsoft.ApiManagement/stable/2018-01-01/APIManagement.json
-- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/1b0e82df2f7f953796af2aee0372e34e00baa2d2/specification/eventgrid/data-plane/Microsoft.HealthcareApis/stable/2018-01-01/HealthcareApis.json
-- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/1b0e82df2f7f953796af2aee0372e34e00baa2d2/specification/eventgrid/data-plane/Microsoft.DataBox/stable/2018-01-01/DataBox.json
-- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/1b0e82df2f7f953796af2aee0372e34e00baa2d2/specification/eventgrid/data-plane/Microsoft.ResourceNotifications/stable/2018-01-01/common.json
-- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/1b0e82df2f7f953796af2aee0372e34e00baa2d2/specification/eventgrid/data-plane/Microsoft.ResourceNotifications/stable/2018-01-01/HealthResources.json
-- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/1b0e82df2f7f953796af2aee0372e34e00baa2d2/specification/eventgrid/data-plane/Microsoft.ResourceNotifications/stable/2018-01-01/Resources.json
-- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/1b0e82df2f7f953796af2aee0372e34e00baa2d2/specification/eventgrid/data-plane/Microsoft.AVS/stable/2018-01-01/PrivateCloud.json
-- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/1b0e82df2f7f953796af2aee0372e34e00baa2d2/specification/eventgrid/data-plane/Microsoft.ApiCenter/stable/2018-01-01/ApiCenter.json
+- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/0d023bd802cde559706b5865f685a10a88516e09/specification/eventgrid/data-plane/Microsoft.Storage/stable/2018-01-01/Storage.json
```

Comment:
This can be simplified by just using the `$(commit-id)` variable and declaring the `commit-id` variable in the yaml.

```yaml


use: '@autorest/java@4.1.25'
commit-id: 0d023bd802cde559706b5865f685a10a88516e09
java: true
...

input-file: 
- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/$(commit-id)/specification/eventgrid/data-plane/Microsoft.Storage/stable/2018-01-01/Storage.json
- ....
```